src/views/Question.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import { Button } from "react-bootstrap";
+import styled from "styled-components";
+import graphql from 'babel-plugin-relay/macro'
+
+import Query from '../components/Query'
+import QuestionCard from "../components/QuestionCard";
+import SubmitAnswerForm from "../components/SubmitAnswerForm";
+import AnswerCard from "../components/AnswerCard";
+
+const query = graphql`
+  query QuestionQuery ($question_id: ID!){
+    node(id: $question_id) {
+      ...QuestionCard_question
+      ...on QuestionNode {
+        answers { edges { node { id ...AnswerCard_answer}}}
+      }
+    }
+  }
+`
+
+export default function Question(props) {
+  const question_id = props.match.params.id;
+  const [showAnswerForm, setShowAnswerForm] = useState(false)
+
+  return <Query
+    query={query}
+    variables={{ question_id }}
+    render={({ error, props }) => {
+      if (!props) {
+        return <div>Loading...</div>
+      } else if (error) {
+        return <div>{error.message}</div>
+      } else {
+        return <Wrapper>
+          <QuestionCard key={question_id} question={props.node} disableBtmBorader={true} visual />
```

Comment:
I think the disabling of bottom border should be a style related thing, not component logic. You can simply disable the bottom border for the last question card in a list, which should also be true for the listing page. You also have a typo there (Borader).

Reply:
not sure how i can do it without logic. u might want to show how? Else I wont be making changes for this.